test/helpers.js

```diff
@@ -1,3 +1,11 @@
+import nock from 'nock';
+
+export function nodeIsOnline (url) {
```

Comment:
can we rename it to `fakeNodeIsOnline`, `mockNodeIsOnline` or something along the lines?